src/legacy/plugin_discovery/types.ts

```diff
@@ -73,6 +73,8 @@ export interface LegacyPluginOptions {
     visTypes: string[];
     embeddableActions?: string[];
     embeddableFactories?: string[];
+    interpreter?: string[];
```

Comment:
I think `interpreter` property should be removed in future. not sure that we need to add this one into this interface

Reply:
Removed